src/stickybits.js

```diff
@@ -226,11 +226,11 @@ class Stickybits {
       ? p.customStickyChangeNumber
       : el.offsetHeight
     it.offset = scrollElOffset + p.stickyBitStickyOffset
-    it.stickyStart = isBottom ? stickyStart - it.offset : 0
+    it.stickyStart = isTop ? stickyStart - it.offset : 0
     it.stickyChange = it.stickyStart + stickyChangeOffset
-    it.stickyStop = isBottom
+    it.stickyStop = isTop
       ? (stickyStart + parent.offsetHeight) - (it.el.offsetHeight + it.offset)
-      : stickyStart + parent.offsetHeight
+      : stickyStart + parent.offsetHeight - window.innerHeight
```

Comment:
Add parentheses...Maybe even a variable for this so it is more descriptive?

```javascript

it.stickyStop = isTop
      ? (stickyStart + parent.offsetHeight) - (it.el.offsetHeight + it.offset)	      
      : stickyStart + (parent.offsetHeight - window.innerHeight)

```

Something like:

```javascript

// for the ternaries; ~line 221
const calculatedParentScrollArea = parent.offsetHeight - window.innerHeight

// the for it.stickyStop; ~line 231
 it.stickyStop = isTop
      ? (stickyStart + parent.offsetHeight) - (it.el.offsetHeight + it.offset)	      
      : stickyStart + calculatedParentScrollArea

```

Feel free to come up with a better variable name.

Reply:
Tbh, I don't think that extra variable will improve readability - it'll just increase cognitive load. As to parentheses - I've missed that it's in a project style and gladly add them.